core/src/main/resources/jenkins/split-plugin-cycles.txt

```diff
@@ -32,3 +32,32 @@ junit jaxb
 bouncycastle-api jaxb
 command-launcher jaxb
 jdk-tool jaxb
+
+#JENKINS-43610 Split Trilead out from Core
+BlameSubversion trilead-api
```

Comment:
I do not think you understand what this file is for. Revert the changes to this file.

Reply:
I've reverted the changes